src/duration.cpp

```diff
@@ -11,8 +11,12 @@
 #include "pseudovector.hpp"
 #include "utilities.hpp"
 
-Global::duration from_string(const std::string& str) {
-  Global::duration d = std::chrono::seconds(0);
+
+using namespace nanotime;
+
+
+duration nanotime::from_string(const std::string& str) {
```

Comment:
A `using namespace` right above it but for clarity still a `nanotime::` prefix?

Reply:
That's because these functions are external to a class. Without the prefix they get picked up as a new function. Could also enclose them inside `namespace nanotime { }`, but there are only a few functions like that and it saves a couple of lines. Could go the namespace way if you have a preference, I've not got any strong opinion here - both are correct C++.

Reply:
Doh I overlooked the free function part...